auxiliary_files/config_files/mumble-server.logrotate.in

```diff
@@ -0,0 +1,10 @@
+/var/log/@MUMBLE_SERVER_BINARY_NAME@/mumble-server.log {
+	weekly
+	rotate 7
+	compress
+	delaycompress
+	missingok
+	postrotate
+		start-stop-daemon --quiet --oknodo --stop --signal 1 --exec /usr/sbin/@MUMBLE_SERVER_BINARY_NAME@ --user mumble-server --pidfile /var/run/@MUMBLE_SERVER_BINARY_NAME@/mumble-server.pid
```

Comment:
The call to [start-stop-daemon](https://www.man7.org/linux/man-pages/man8/start-stop-daemon.8.html) is not supported by systemd, which means this file (in its current form) has no meaning on all systemd based Linux distributions and will only work on those using sysvinit style init systems (see https://en.wikipedia.org/wiki/Init#System_V).

It might be best to not install this file to the target system automatically, as it would malfunction on systemd based OSes.